abopt/base.py

```diff
@@ -323,6 +358,11 @@ def g(self, x):
         g = self._gradient(x)
         return g
 
+    def f_g(self, x):
```

Comment:
Perhaps just call it fg? f_g may imply `$f_g$`. No strong preference here.

Reply:
done